.kontinuous/env/prod/values.yaml

```diff
@@ -38,11 +38,11 @@ hasura:
     enabled: true
   resources:
     limits:
-      cpu: "2000m"
-      memory: "4Gi"
-    requests:
-      cpu: "1000m"
+      cpu: "500m"
       memory: "1Gi"
```

Comment:
Ah oui, pas possible dans le cas où on lance un glossaire

Reply:
@igorrenquin on peut laisser 4Gi pour la limite ? C'est un service qui peut être assez gourmand sur certaines tâches que l'on réalise très peu mais qui peu arriver.

Reply:
https://grafana-ovh.fabrique.social.gouv.fr/d/ada6934f-a080-4fd9-919d-f9c8a621dec9/tableau-de-bord-code-du-travail-numerique?orgId=1&from=now-30d&to=now

Reply:
Donc oui, c'est sur `hasura` pas sur `export`, donc c'est parfait